doc updates (#313)
regionmask/regionmask: docs/index.rst

create masks of geographical regions
************************************

Determine which geographic region each grid point belongs to.

When working with gridded data such as climate model output or reanalysis data it is

> often important to create regional averages, such as over countries, continents or

Author: ```suggestion
often important to create regional averages, e.g., over countries, continents or
```
